sharing_portal/templates/sharing_portal/share.html

```diff
@@ -86,27 +66,12 @@ <h2 class="artifactTitle">Edit sharing settings</h2>
       </div>
 
       <div class="layoutGrid__side">
-        <div class="artifactVersions">
-          <h4><i class="fa fa-files-o"></i> Versions</h4>
-          {{ z_management_form }}
-          {% for i, form in z_forms %}
-          <fieldset>
-            <span class="artifactVersion__title">
-              <span>Version {{ i }}</span>
-              <span>{{ form.model.created_at }}</span>
-            </span>
-            {% bootstrap_form form form_group_class='form-group artifactVersion__doi' error_css_class='foobar' show_label=False layout='inline' %}
-          </fieldset>
-          {% endfor %}
-        </div>
       </div>
     </div>
   </form>
 
 </div>
 
-{# TODO: replace with json_script filter when we're on Django 2.1 #}
-<script id="artifact-data" type="application/json">{{ artifact_json|safe }}</script>
```

Comment:
I forget what this was even for. I think maybe at some point I had an idea that we would have a Vue form or something, but that didn't work out.